Sources/Extensions/MapKit/MKMapView+PovioKit.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  MKMapView+PovioKit.swift
+//  PovioKit
+//
+//  Created by Borut Tomažin on 11/11/2020.
+//  Copyright © 2020 Povio Labs. All rights reserved.
+//
+
+import MapKit.MKMapView
+
+public extension MKMapView {
+  /// Current zoom level
+  var zoom: Double {
+    log2(360 * ((Double(frame.size.width) / 128) / region.span.longitudeDelta))
+  }
+  
+  /// Returns radius distance in meters from map center to the top of the visible screen
+  var visibleRadius: Double {
+    let centerLocation = CLLocation(latitude: centerCoordinate.latitude, longitude: centerCoordinate.longitude)
+    let topCenterCoordinate = convert(.init(x: bounds.size.width / 2.0, y: 0), toCoordinateFrom: self)
```

Comment:
We could use bounds.midX instead of bounds.size.width / 2.0 🤔

Reply:
Good point 👌